NEWS.md

```diff
@@ -1,5 +1,9 @@
 # volcalc (development version)
 
+# volcalc 1.0.2
```

Comment:
```suggestion

```

I'll do this in the same commit that increments the version in DESCRIPTION and the citation files